scaledimage.cpp

```diff
@@ -85,57 +85,57 @@ void ScaledImage::scale(double factor, double angle) {
 	if (this->isScaled) {
 		SDL_FreeSurface(this->scaledImage);
 		this->scaledImage=NULL;

```

Comment:
This line is superfluous.

Reply:
May be rm after merging, did not introduce that assign.
@szymor I'm working on a release on my dowstream fork, hope u dont mind

Reply:
I know you did not, but you added another line below, which makes this line not needed anymore.

Feel free to work on your fork, ping me if you finish.

Reply:
seems I missed the whole point of isScaled functionality in `scale` member, I'll make another approach